compose/dm-store.yml

```diff
@@ -69,6 +69,38 @@ services:
     networks:
       - ccd-network
 
+  ccd-case-document-am-api:
+    image: "${CCD_CASE_DOCUMENT_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/case_document_am_api:${CCD_CASE_DOCUMENT_API_TAG:-latest}"
+    environment:
+      CCD_DOCUMENT_API_IDAM_KEY: "${IDAM_KEY_CASE_DOCUMENT}"
+      CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor
+      IDAM_API_URL: http://idam-api:5000
+      IDAM_OIDC_URL: http://idam-api:5000
+      S2S_URL: http://service-auth-provider-api:8080
+      IDAM_USER_URL: http://idam-api:5000
```

Comment:
Should this allow calls to stub service?  e.g. `"${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"`

Reply:
+1

Reply:
Fixed in 889a833